SwagLyricsBot/lyrics.py

```diff
@@ -11,7 +11,8 @@ async def fetch(session, url, **kwargs):
     """
     Uses aiohttp to make http GET requests
     """
-    headers = {'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/84.0.4147.89 Safari/537.36'}
+    headers = {'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) '
```

Comment:
Is the space at the end of this string intentional?

Reply:
yeah I just made it go across lines so it's less than 120 chars per line